### PR TITLE
739: rename background colors

### PR DESCRIPTION
**Pull request by bozmen**
### Contributor checklist

- [] This pull request is on a [separate branch](https://docs.github.com/en/get-started/quickstart/github-flow) and not the main branch

---

### Related issue

- #739

### CODE REVIEW

**Comment by andrewtavis** [comment]
Hey @bozmen 👋 I'm a bit worried about the merge conflict here. Let me bring some things in and we can see about it, but maybe I'll ask you to close this one and send it along again after pulling new changes. I'll write in again after I've merged some things and we can see what the conflict situation is like :)

**Comment by andrewtavis** [comment]
We should be able to figure it out though regardless, so if you're on a role feel free to send along the rest and we can fix the conflicts. I think that the changes in other files are pretty local, so it shouldn't be too bad :)

**Comment by bozmen** [comment]
@andrewtavis Alright, was just checking things out. Let me know when you pull in the other things then, then I will continue.

**Comment by andrewtavis** [comment]
Minor merge conflicts fixed in b5623f8 as the new versions of the language and theme selectors are inheriting from a common component 😊 One more PR to merge and then we'll be good for further changes :)

**Comment by andrewtavis** [comment]
@to-sta, it looks like we have a conflict in formatters for something here 🤔

```ts
  return props.isButton
    ? {
        is: "p",
        class: "pl-2 pr-2",
      }
    : props.isSideLeftMenu
    ? {
        is: "p",
        class:
          "select-none whitespace-nowrap hover:light-menu-selection dark:hover:dark-menu-selection",
      }
    : props.iconName
    ? {
        is: "p",
        class: "pl-2 pr-2",
      }
    : {
        is: "span",
      };
```

is what precommit fixes it to, but prettier is expecting all lines after `: props.isSideLeftMenu` to be indented once.

**Comment by andrewtavis** [comment]
Hey @bozmen 👋 All the frontend issues are now merged and the branch has been updated with the current version of main 😊 Feel free to pull the changes down and we can get this through! 🚀

**Comment by bozmen** [comment]
@andrewtavis When I pull/fetch upstream and try to resolve the conflicts there, I get different conflicts than what is reported here. Do not really know why, trying to get used to forking :D

**Comment by andrewtavis** [comment]
Ya there's something a bit screwy with Prettier and our other checks I feel like 🤔 We can maybe make an issue for it. Likely something in the settings for precommit or eslint needs to get updated. Don't stress too much about it, @bozmen!

**Comment by andrewtavis** [comment]
Quick note on this, @bozmen: the issue with formatting is that pre-commit is using a default configuration of Prettier that maybe has a lower version than the extension, so we're saving files and the extension is fixing them, but the pre-commit version is saying they're wrong. We can basically just run pre-commit so that it's fixed and then not save the file afterwards :) Don't worry about this though 😊

**Comment by andrewtavis** [comment]
87b86b2 fixes the issue by updating the Prettier version for pre-commit 😊 Thanks @to-sta!

File: frontend/components/card/discussion/CardDiscussionInput.vue [hunk 1]
```diff
@@ -97,7 +97,7 @@
         <textarea
           id="message"
           rows="4"
-          class="block p-2.5 w-full text-sm text-light-text rounded-lg border border-light-section-div placeholder-light-distinct-text dark:placeholder-dark-distinct-text dark:text-dark-text dark:bg-dark-distinct focus-brand"
+          class="block p-2.5 w-full text-sm text-light-text rounded-lg border border-light-section-div placeholder-light-distinct-text dark:placeholder-dark-distinct-text dark:text-dark-text dark:bg-dark-layer-1 focus-brand"
```
Comment (andrewtavis) [inline frontend/components/card/discussion/CardDiscussionInput.vue:100]: Looks like this is missing light mode bg and a dark mode border. I can get this 😊

**Review by andrewtavis — APPROVED** [review]
Thanks for the discussion about all this and the changes, @bozmen! Great to have you on the team! 😊😊🥳

**Comment by bozmen** [comment]
@andrewtavis No worries! Sorry did not have time to look at the convo here yet. Did we want to change some other variable names as well as far as I recall? If not with this issue, then we can discuss further more

File: frontend/components/card/discussion/CardDiscussionInput.vue (hunk 1 above)
Comment (bozmen) [reply frontend/components/card/discussion/CardDiscussionInput.vue]: Oh right, good side effects of going through basically the whole code :P

**Comment by bozmen** [comment]
> Quick note on this, @bozmen: the issue with formatting is that pre-commit is using a default configuration of Prettier that maybe has a lower version than the extension, so we're saving files and the extension is fixing them, but the pre-commit version is saying they're wrong. We can basically just run pre-commit so that it's fixed and then not save the file afterwards :) Don't worry about this though 😊

I still did not configure it for myself, probably this is a good time also after fix for the said problem

**Comment by andrewtavis** [comment]
We can make another issue, @bozmen :) Let me make one and I'll tag you to work on it!

**Pull request merged**